### ISSUE TITLE
The problem of training StyleAvatar from scratch

### ISSUE THREAD

**Issue by huangyongbobo** [LizhenWangT/StyleAvatar#38]
Thank you for your excellent work and open-source code. I am trying to train a model from scratch using my own dataset. During the training process, I noticed that the results in the "sample" folder were very good. However, when I used your default "test.py" file for inference with the trained model, I obtained poor results. Therefore, I would like to seek your advice on where the problem might be. Thank you very much, and best wishes.

**Comment by Fritskee** [comment]
@huangyongbobo did you manage to get good results with StyleAvatar on a self-trained model? I'm really struggling hard to get good results. Would like your advice in this case.

**Comment by huangyongbobo** [comment]
> @huangyongbobo did you manage to get good results with StyleAvatar on a self-trained model? I'm really struggling hard to get good results. Would like your advice in this case.

Yes, I did it, it's amazing. Maybe multiple aspects: data(main), pre-training way, loss and code adjustments.

**Comment by Fritskee** [comment]
> > @huangyongbobo did you manage to get good results with StyleAvatar on a self-trained model? I'm really struggling hard to get good results. Would like your advice in this case.
> 
> Yes, I did it, it's amazing. Maybe multiple aspects: data(main), pre-training way, loss and code adjustments.

Thanks so much for your reply! Would you mind sharing a bit more what you mean by that?

1. What "changes" to your data allowed you to obtain great results? 
2. What did you change to the pre-training?
3. Which loss function did you use instead?
4. What other changes were necessary to the code?

**Comment by huangyongbobo** [comment]
> > > @huangyongbobo did you manage to get good results with StyleAvatar on a self-trained model? I'm really struggling hard to get good results. Would like your advice in this case.
> > 
> > 
> > Yes, I did it, it's amazing. Maybe multiple aspects: data(main), pre-training way, loss and code adjustments.
> 
> Thanks so much for your reply! Would you mind sharing a bit more what you mean by that?
> 
> 1. What "changes" to your data allowed you to obtain great results?
> 2. What did you change to the pre-training?
> 3. Which loss function did you use instead?
> 4. What other changes were necessary to the code?

1) For video frame, pay attention to the richness of facial expressions and head postures, as well as mouth movements. For data preprocessing, parameter optimization can also be attempted.
2) Rewrite pre-trained scripts.
3) The trade-off between vgg loss  and GAN loss, as they may lead to conflicts, can be adjusted to slow down the epoch strategy during training.
4) Some detail adjustments can be sensed but may not expressed in words...

**Comment by zain3211** [comment]
> Thank you for your excellent work and open-source code. I am trying to train a model from scratch using my own dataset. During the training process, I noticed that the results in the "sample" folder were very good. However, when I used your default "test.py" file for inference with the trained model, I obtained poor results. Therefore, I would like to seek your advice on where the problem might be. Thank you very much, and best wishes.

hello sir please drop you whatsapp number or any social media link i need your help
